packages/docusaurus-theme-classic/src/theme/DocSidebarItem/styles.module.css

```diff
@@ -6,18 +6,6 @@
  */
 
 @media (min-width: 997px) {
-  .menuLinkText {
-    cursor: initial;
```

Comment:
If the `href` attribute is missing in a link, the default cursor is used, so there is no need for this class.